statesman-publisher/pom.xml

```diff
@@ -19,6 +19,12 @@
     </properties>
 
     <dependencies>
+        <dependency>
```

Comment:
This dependency is pulled from some weird github repo. Can you replace with this?
```
<dependency>
    <groupId>com.baidu</groupId>
    <artifactId>leansoft-bigqueue</artifactId>
    <version>0.7.3</version>
</dependency>
```